script.js

```diff
@@ -1,4 +1,151 @@
+    + randomDice1 + ' for dice 1 and ' + randomDice2 + ' for dice 2 <br> choose the order of the dice';
+  return message;
+};
+
+// player one pick dice logic
+var playerOnePickDiceOneOrDiceTwo = function (input) {
+  var message = '';
+  playerOneHighestNum = 0;
+  // If player 1 pick dice 1
+  if (input == '1') {
+    playerOneHighestNum = randomDice1 + '' + randomDice2;
+    message = 'Player 1, you chose Dice 1 first.<br> Your number is ' + playerOneHighestNum + '. <br> It is now player 2 turn';
+    console.log('player 1');
+    console.log(playerOneHighestNum);
+    return message;
+  }
+  // if player 1 pick dice 2
+  if (input == '2') {
+    playerOneHighestNum = randomDice2 + '' + randomDice1;
+    message = 'Player 1, you chose Dice 2 first.<br> Your number is ' + playerOneHighestNum + '. <br> It is now player 2 turn';
+    console.log('player 1');
+    console.log(playerOneHighestNum);
+    return message;
+  }
+};
+
+// player two roll 2 dice logic
+var rollTwoDiceForPlayerTwo = function () {
+  var message = '';
+  randomDice1 = 0;
+  randomDice2 = 0;
+  randomDice1 = diceRoll1();
+  randomDice2 = diceRoll2();
+  console.log(randomDice1);
+  console.log(randomDice2);
+  message = 'Welcome player 2.<br> You rolled '
+     + randomDice1 + ' for dice 1 and ' + randomDice2 + ' for dice 2 <br> choose the order of the dice';
+  return message;
+};
+// player two pick dice logic
+var playerTwoPickDiceOneOrDiceTwo = function (input) {
+  var message = '';
+  var playerTwoHighestNum = 0;
+  // If player 2 pick dice 1
+  if (input == '1') {
+    playerTwoHighestNum = randomDice1 + '' + randomDice2;
+    console.log('player 2 ');
+    console.log(playerTwoHighestNum);
+    console.log('Player 1 ');
+    console.log(playerOneHighestNum);
+    // If player 1 combine number is higher than player 2 combine number.
+    // player 1 win. Else player 2 win.
+    if (playerOneHighestNum > playerTwoHighestNum) {
+      message = 'Player 2, you chose Dice 1 first.<br> You number '
+     + playerTwoHighestNum + '.<br> Player 1 win ';
+      return message;
+    }
+    message = 'Player 2, you chose Dice 1 first.<br> You number '
+     + playerTwoHighestNum + '<br>Player 2 win ';
+    return message;
+
+    // If player 2 pick dice 2
+  } if (input == '2') {
+    playerTwoHighestNum = randomDice2 + '' + randomDice1;
+    console.log('player 2 ');
+    console.log(playerTwoHighestNum);
+    console.log('player 1');
+    console.log(playerOneHighestNum);
+    // If player 2 combine number is higher than player 1 combine number.
+    // player 2 win. Else player 1
+    if (playerTwoHighestNum > playerOneHighestNum) {
+      message = 'Player 2, you chose Dice 2 first.<br> You number is '
+      + playerTwoHighestNum + '<br> Player 2 win ';
+      return message;
+    }
+    message = 'Player 2, you chose Dice 2 first.<br> You number is '
+    + playerTwoHighestNum + '<br> Player 1 win ';
+    return message;
+  }
+};
+
 var main = function (input) {
```

Comment:
Great work keeping the main function clean. Please re-use your roll and pick dice function logic and avoid having a separate helper function for the same logic for each player!